assets/scss/dropdown.scss

```diff
@@ -136,15 +138,15 @@
             padding: 0.25rem 1rem;
           }
 
-          input[type="checkbox"] {
+          input[type="checkbox"],input[type="radio"] {
```

Comment:
For consistency (and readability) with other CSS files, move `input[type="radio"]` to the next line.